### DISCUSSION TITLE
Had to re-install add-on - now broken

### DISCUSSION THREAD

**Discussion by itm1960** [t0bst4r/home-assistant-matter-hub#319] [Q&A] [answered]
Due to a corrupt HAOS install, I needed to uninstall and re-install the Matter Hub add-on. My main usage is to execute HA scripts, which are listed as switches in the add-on). While this was working before re-installing the add-on, I cannot get it to work again. There are now duplicates of all of my script/switches in the Google Home app, but assigning either of them to an automation doesn't do anything.
When I uninstalled the add-on I chose the "remove data" option, and I re-scanned the QR code when re-installing.
Any idea how I can re-activate my shared devices/scripts/switches?

**Comment by t0bst4r** [comment, marked as answer]
When you removed the data when uninstalling, you can only restore it with a HA backup. You could also try to extract the HA backup and only import the files in the hamh addon directory. But if you already removed the working bridge from google home, you'll need to re-pair anyway. 

When trying again, I'd recommend deleting all hamh devices first from google home.

E.g. I've seen devices not being deleted in Alexa, too. I needed to delete them by hand.

**Reply by itm1960** [reply]
Thanks for your reply. I eventually managed to remove the devices from the Google Home app, then  re-added everything.
